src/main/charts/bitbucket/templates/NOTES.txt

```diff
@@ -13,4 +13,20 @@ To see the custom values you used for this release:
 
   $ helm get values {{ .Release.Name }} -n {{ .Release.Namespace }}
 
+{{ if .Values.ingress.create -}}
+Service URL of {{ .Chart.Name }}: {{ ternary "https" "http" .Values.ingress.https -}}://{{ .Values.ingress.host }}
+{{- else if contains "ClusterIP"  .Values.bitbucket.service.type -}}
```

Comment:
What if they don't use `ClusterIP` type nor Ingress? Should we cater for other types as well?
https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types

It seems it could be possible: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/templates/NOTES.txt#L12

Reply:
I excluded `NodePort` and `LoadBalancer` because these two types are not appearing anywhere in `Values.<product>.service.type`, hence I thought will not go into these conditions anyway. Or did I misunderstand here?